9.4/Dockerfile.rhel7

```diff
@@ -34,7 +35,7 @@ RUN yum install -y yum-utils gettext && \
     yum-config-manager --enable rhel-server-rhscl-7-rpms && \
     yum-config-manager --enable rhel-7-server-optional-rpms && \
     yum install -y --setopt=tsflags=nodocs bind-utils rh-postgresql94 && \
-    yum install -y --disablerepo="epel" --setopt=tsflags=nodocs nss_wrapper && \
```

Comment:
@praiskup this is needed for the jenkins build... we have epel repo enabled on the host when we build and we don't want to install the nss_wrapper from epel (we want to install from official channels). This will prevent yum using EPEL.